include/boost/geometry/algorithms/detail/overlay/traversal.hpp

```diff
@@ -569,28 +569,24 @@ public :
         return true;
     }
 
-
-    template <typename RankedPoint>
-    inline turn_operation_type const& operation_from_rank(RankedPoint const& rp) const
-    {
-        return m_turns[rp.turn_index].operations[rp.operation_index];
-    }
-
     inline int select_turn_in_cluster_union(sort_by_side::rank_type selected_rank,
             typename sbs_type::rp const& ranked_point,
+            std::set<signed_size_type> const& cluster_indices,
             signed_size_type start_turn_index, int start_op_index) const
     {
-        // Returns 0 if it not OK
-        // Returns 1 if it OK
-        // Returns 2 if it OK and start turn matches
-        // Returns 3 if it OK and start turn and start op both match
+        // Returns 0: not OK
+        // Returns 1: OK but next turn is in same cluster
+        // Returns 2: OK
+        // Returns 3: OK and start turn matches
+        // Returns 4: OK and start turn and start op both match
```

Comment:
Why not make an enum describing this? These values are not used to calculate something are they?

Reply:
I'm not sure about that. We just need an order here, and we don't have to name the states.
The comments are to explain the ordering and only relevant for this place, and no other place only.
If it would be an enum (possible of course) we would only assign it and never use it, and it would be more error-prone if people accidentally change the order later (or we should assign values to them)

Reply:
Ah ok, I see now that this function is called below in `select_from_cluster_union` and what is returned here is some kind of priority. So indeed an enum is not needed because the caller is not concerned about specific values, only about relative priority.

But AFAIU this means that even though this function is called `select_turn_in_cluster_union` it doesn't select anything. It calculates this priority value instead. If my understanding is correct I propose to rename it accordingly. To e.g. `priority_of_turn_in_cluster_union()` or whatever you think reflects values that are returned the best.

Reply:
Thanks, good idea, `priority...` is good, I will rename the function

Reply:
:heavy_check_mark: